examples/java/com/ibm/watson/developer_cloud/document_conversion/v1/DocumentConversionIndexDocumentExample.java

```diff
@@ -148,6 +161,18 @@ public static void main(String[] args) {
         .build();
     String indexDocumentWithMediaTypeAndMetadataAndConfig = service.indexDocument(indexDocumentOptions10).execute();
     System.out.println(indexDocumentWithMediaTypeAndMetadataAndConfig);
+
+    printHeaderForExample("Index document with provided media type, metadata, convert document config, and fields");
+    IndexDocumentOptions indexDocumentOptions11 = new IndexDocumentOptions.Builder()
+        .document(document)
+        .metadata(metadata)
+        .mediaType(HttpMediaType.TEXT_HTML)
+        .convertDocumentConfig(convertDocumentConfig)
+        .indexConfiguration(indexConfigurationWithFields)
+        .dryRun(false)
```

Comment:
`dryRun` should be false by defaults so that this line doesn't need to be required.